### DISCUSSION TITLE
Add custom translation assist

### DISCUSSION THREAD

**Discussion by BobSanScans** [xulihang/ImageTrans-docs#465] [Q&A]
Is it possible to add your own translation assist/parameters; 
I have found solid translation success with RomajiDesu for Japanese to English, and it can be easily used/integrated by calling:
https://www.romajidesu.com/translator/{insert Japanese text here}

Is there a way I can manually add this as an option for it to run or does it have to by added into the base software itself?

**Comment by xulihang** [comment]
It is the same as this discussion about the web machine translation plugin: https://github.com/xulihang/ImageTrans-docs/discussions/463
